packages/cloudvision-grpc-web/src/grpc/GrpcSubjectWrapper.ts

```diff
@@ -0,0 +1,37 @@
+import { Observable, Subject, Subscription } from 'rxjs';
+
+export default class GrpcSubjectWrapper<T> extends Observable<T> {
```

Comment:
This is not the right approach.
Whenever you see yourself writing an observable "wrapper", you really should consider that an operator would be a better choice.
In this case, you're trying to run a cleanup function on unsubscribe? That's the `finalize` operator.

Reply:
Yes but cleanup only when there is no one listening